src/test/java/com/uber/cadence/workflow/MetricsTest.java

```diff
@@ -108,21 +104,79 @@ public void executeChild() {
     }
   }
 
-  @Before
-  public void setUp() {
+  public interface ReceiveSignalObject_ChildWorkflow {
```

Comment:
Java convention is to use CamelCase for Class names. No underscores allowed.http://www.oracle.com/technetwork/java/codeconventions-135099.html